src/app/modules/spcp/spcp.controller.ts

```diff
@@ -267,6 +267,8 @@ export const appendVerifiedSPCPResponses: RequestHandler<
       req.body.parsedResponses.push(...createSingpassParsedResponses(uinFin))
       break
     case AuthType.CP:
+      // Note that maskUidOnLastField() relies on the fact that userInfo is pushed in last to parsedResponses
+      // TODO: Remove this after refactor in #1104 (https://github.com/opengovsg/FormSG/issues/1104)
```

Comment:
FYI, there is a GitHub VSCode extension called "GitHub Pull Requests and Issues" that should be able to pull the issue details if you write the TODO in this manner
```suggestion
      // TODO(#1104): Remove this after refactor
```